partner_account_payment_mode_multicompany_default/models/partner.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+from odoo import models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    def _get_multicompany_account_payment_mode_key(self):
+        return "name"
```

Comment:
Ask : shall return the "Code" ?

Reply:
Payment mode is mapped by name inter-company. There's no code field